src/parser/core/modules/AdditionalEvents.js

```diff
@@ -43,6 +45,7 @@ export default class AdditionalEvents extends Module {
 	static dependencies = [
 		'enemies',
 		'data',
+		'normalisedEvents', // eslint-disable-line @xivanalysis/no-unused-dependencies
```

Comment:
My two reasons for adding the dependency were to:
1) Ensure the normaliser for normalisedEvents ran, without requiring implementers to specifically add the dependency (under the mental image that this would be a hard replacement for aoedamage and the getFFLogsEvents compatibility shim)
2) Ensure the fabricated events got sorted in with the other events, without having to deal with a bunch of individual splicing when dumping the events out (doubly so since splicing in the middle of Array.forEach requires some funky handling to avoid infinitely looping on the added event).